packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -76,22 +91,25 @@
 }
 
 .divider-container {
-  @apply flex items-stretch border-color-input
+  @apply flex items-stretch
   border
   border-l-0
   border-r-0
   border-solid;
-  background-color: var(--calcite-color-foreground-1);
+  background-color: var(--calcite-input-date-picker-background-color, var(--calcite-color-foreground-1));
+  border-color: var(--calcite-input-date-picker-border-color, var(--calcite-color-border-input));
 }
 
 :host([layout="horizontal"]) .divider-container {
-  @apply w-px;
+  inline-size: var(--calcite-spacing-px);
 }
 
 .divider {
-  @apply inline-block w-px;
+  @apply inline-block;
+
   margin-block: var(--calcite-spacing-xxs);
-  background-color: var(--calcite-color-border-2);
+  background-color: var(--calcite-input-date-picker-background-color, var(--calcite-color-border-2));
```

Comment:
I don't think `--calcite-input-date-picker-background-color` is the right one here as it doesn't configure background-color for inputs or floating-ui of input-date-picker. Perhaps 
`--calcite-input-date-picker-input-divider-background-color` might work. Thoughts?